extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts

```diff
@@ -63,25 +77,29 @@ export class InstanceSummary {
 			value: "",
 			CSSStyles: {
 				...styles.LIGHT_LABEL_CSS,
-				'margin': '0px',
-				'padding-left': '10px'
+				'margin': '0px'
 			}
 		}).component();
 
-		instanceSummaryContainer.addItems([description, this._assessedDatabases, this.createGraphComponent(),
-			findingsSummaryTitle, this._totalFindingLabels]);
+		this._tdeInfoContainer = await this.createTdeInfoContainer();
+
+		instanceSummaryContainer.addItems([description,
+			this._assessedDatabases, this.createGraphComponent(),
+			this._tdeInfoContainer, findingsSummaryTitle,
+			this._totalFindingLabels]);
 
 		return instanceSummaryContainer;
 	}
 
 	// function to populate instance summary container with latest values.
-	public populateInstanceSummaryContainer(model: MigrationStateModel): void {
-
-		this._assessedDatabases.value = constants.ASSESSED_DBS_LABEL + ": " + model._databasesForAssessment?.length;
-		this._totalFindingLabels.value = constants.TOTAL_FINDINGS_LABEL + ": " + model._assessmentResults?.issues.filter(issue => issue.appliesToMigrationTargetPlatform === model._targetType).length;
-		const readyDbsCount = model._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === model._targetType).length === 0).length;
-		const notReadyDbsCount = model._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === model._targetType && issue.issueCategory === IssueCategory.Issue).length !== 0).length;
-		const readyWithWarnDbsCount = model._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === model._targetType && issue.issueCategory === IssueCategory.Warning).length !== 0).length;
+	public async populateInstanceSummaryContainer(): Promise<void> {
+		this._previousMiTdeMigrationConfig = this.migrationStateModel.tdeMigrationConfig;
+		await this.refreshTdeView();
+		this._assessedDatabases.value = constants.ASSESSED_DBS_LABEL + ": " + this.migrationStateModel._databasesForAssessment?.length;
+		this._totalFindingLabels.value = constants.TOTAL_FINDINGS_LABEL + ": " + this.migrationStateModel._assessmentResults?.issues.filter(issue => issue.appliesToMigrationTargetPlatform === this.migrationStateModel._targetType).length;
+		const readyDbsCount = this.migrationStateModel._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === this.migrationStateModel._targetType).length === 0).length;
+		const notReadyDbsCount = this.migrationStateModel._assessmentResults.databaseAssessments.filter((db) => db.issues.filter(issue => issue.appliesToMigrationTargetPlatform === this.migrationStateModel._targetType && issue.issueCategory === IssueCategory.Issue).length !== 0).length;
+		const readyWithWarnDbsCount = this.migrationStateModel._databasesForAssessment?.length - (readyDbsCount + notReadyDbsCount);
```

Comment:
this.migrationStateModel._assessmentResults.databaseAssessments is used multiple times. We can store this in a separate variable say totalDbs and use it again.

Reply:
done